README.md

```diff
@@ -20,12 +20,21 @@ or actually inside the augur directory running
 
 This step-by-step installaion shows you how to get a working environment with `firecrown` and `augur` that you can hack away efficiently.
 
+### Create a new conda environment
 Start by creating a new anaconda environment:
 
-```
-conda create --name forecasting
+```bash
+conda env create --name forecasting --file=environment.yml
 conda activate forecasting
 ```
+### Or updaing an existing environment
+If you already have an existing environment, you can update it with the following command:
+
+```bash
+conda activate my_env
+conda env update --name my_env --file=environment.yml --prune
+```
+and activate your environment with `conda activate [my_env/forecasting]`.
 
 Next install firecrown and augur.
```

Comment:
The conda environment created by the previous `conda env create` command already includes Firecrown. Unless the Augur developers are planning to also modify Firecrown code, there is no reason to clone the Firecrown repository and then to pip-install Firecrown into the environment a second time.

For those Augur developers who *do* want to modify Firecrown code, it would be much better to have another YAML file to create an environment that does *not* contain Firecrown (but that does contain all of the things upon which Firecrown depends). That is complicated; in Firecrown we provide such an environment file.

Reply:
@marcpaterno if this is about lines 34-35: this was supposed to be an alternative for updating an env the user has, instead of creating a new one.

The `Next install firecrown` was there before, but I agree that it should be modified to reflect that this step is only needed if the user wants to modify firecrown. I will update the readme to reflect this, if you agree!

Reply:
@arthurmloureiro I agree with removing the instructions to clone Firecrown and then pip-install it. But I was also pointing out that if someone working with Augur *does* want to modify Firecrown code, they should *not* also have Firecrown (with an older version) already installed in the conda environment into which the new pip-install will be done. It can be quite tricky to make sure that such an environment has consistent versions of everything to work together correctly.